freeipa-api/src/main/java/com/sequenceiq/freeipa/api/v1/freeipa/stack/model/common/Status.java

```diff
@@ -38,13 +38,18 @@ public enum Status {
     // CCM upgrade statuses
     UPGRADE_CCM_REQUESTED,
     UPGRADE_CCM_IN_PROGRESS,
-    UPGRADE_CCM_FAILED;
+    UPGRADE_CCM_FAILED,
+    // Modify proxy config statuses
```

Comment:
unnecessary comment

Reply:
same as for ccm, I think it acts as a divider that these statuses belong together

Reply:
that could be removed too, name itself